stan/math/prim/fun/hypergeometric_3F2.hpp

```diff
@@ -109,7 +111,8 @@ T_return hypergeometric_3F2_infsum(const Ta& a, const Tb& b, const Tz& z,
  * @param[in] z z (is always called with 1 from beta binomial cdfs)
  * @param[in] precision precision of the infinite sum. defaults to 1e-6
  * @param[in] max_steps number of steps to take. defaults to 1e5
- * @return Generalized hypergeometric function applied to the inputs
+ * The 3F2 generalized hypergeometric function applied to the
```

Comment:
just add back `@return`

Reply:
Ahh the dangers of blindly copy-pasting! Thanks will fix those now